src/pip/_internal/download.py

```diff
@@ -119,7 +137,10 @@ def user_agent():
         data["cpu"] = platform.machine()
 
     if HAS_TLS:
-        data["openssl_version"] = ssl.OPENSSL_VERSION
+        if IS_PYOPENSSL:
+            data["openssl_version"] = OpenSSL.SSL.OPENSSL_VERSION_NUMBER
```

Comment:
This is wrong:

```
>>> import OpenSSL
>>> OpenSSL.SSL.OPENSSL_VERSION_NUMBER
269484175
>>> import ssl
>>> ssl.OPENSSL_VERSION
'OpenSSL 1.0.2o  27 Mar 2018'
```

Reply:
I actually believe we should roll this back entirely and always send `ssl.OPENSSL_VERSION`, so the data is consistent. Sending different data in ways that can't be detected in teh database is a bad idea.

Reply:
Yea, after talking to Alex I think I agree.

Reply:
Sure.

Reply:
If there's some way to make a string representation of the version number, would that be fine?